ui/modal/modalRepost/view.jsx

```diff
@@ -0,0 +1,199 @@
+
+  const myClaimsString = myClaims && myClaims.map(channel => channel.permanent_url).join(',');
+  React.useEffect(() => {
+    if (myClaimsString === '') {
+      doFetchClaimListMine();
+    }
+  }, [myClaimsString, doFetchClaimListMine]);
+
+  function handleSubmit() {
+    const channelToRepostTo = channels && channels.find(channel => channel.name === repostChannel);
+    if (channelToRepostTo && repostName && repostBid && repostChannel && contentClaimId) {
+      doRepost({
+        name: repostName,
+        bid: creditsToString(repostBid),
+        channel_id: channelToRepostTo.claim_id,
+        claim_id: contentClaimId,
+      }).then(() => {
+        doHideModal();
+        doToast({ message: __('Woohoo! Sucessfully reposted this claim.') });
+      });
+    }
+  }
+
+  function handleCloseModal() {
+    doClearRepostError();
+    doHideModal();
+  }
+
+  const showAdvancedSection = showAdvanced || repostNameError || repostBidError;
+  return (
+    <Modal isOpen type="card" onAborted={handleCloseModal} onConfirmed={handleCloseModal}>
+      <Card
+        actionIconPadding={false}
+        icon={ICONS.REPOST}
+        title={
+          <span>
+            Repost <em>{title}</em>
+          </span>
+        }
+        subtitle={
+          error ? (
+            <ErrorText>{__('There was an error reposting this claim. Please try again later.')}</ErrorText>
+          ) : (
+            <span>{__('Repost your favorite claims to help more people discover them!')}</span>
+          )
+        }
+        actions={
+          <div>
+            <SelectChannel
+              label="Channel to repost on"
+              hideAnon
+              channel={repostChannel}
+              onChannelChange={newChannel => setRepostChannel(newChannel)}
+            />
+            <div className="section__actions">
+              {!showAdvancedSection && (
+                <Button button="link" label={__('Advanced')} onClick={() => setShowAdvanced(true)} />
+              )}
+            </div>
+            {showAdvancedSection && (
+              <React.Fragment>
+                <fieldset-group class="fieldset-group--smushed fieldset-group--disabled-prefix">
+                  <fieldset-section>
+                    <label>{__('Name')}</label>
+                    <div className="form-field__prefix">{`lbry://${
+                      !repostChannel || repostChannel === CHANNEL_NEW ? '' : `${repostChannel}/`
+                    }`}</div>
+                  </fieldset-section>
+                  <FormField
+                    type="text"
+                    name="repost_name"
+                    value={repostName}
+                    error={repostNameError}
+                    onChange={event => setRepostName(event.target.value)}
+                  />
+                </fieldset-group>
+                <div className="form-field__help">
+                  {__('The name of your repost, something about reposting to help search')}
```

Comment:
Full text: Change this to repost to a different [community name](https://lbry.com/faq/naming).